#  TODO: FIX This all node behaviour -> check if it's a Foxy issue
# /**/differential_flatness_controller: 
/**:
  ros__parameters:
    uav_mass: 1.4
    antiwindup_cte : 1.0
    trajectory_following:
      position_Kp:
        x: 6.0
        y: 6.0
        z: 6.0
      position_Ki:
        x: 0.01
        y: 0.01
        z: 0.01
      speed_Kp:
        x: 3.0
        y: 3.0
        z: 3.0
    speed_following:
      speed_Kp: 
        x: 3.0
        y: 3.0
        z: 4.0
      speed_Kd: 
        x: 0.0
        y: 0.0
        z: 0.0
      speed_Ki:
        x: 0.0
        y: 0.0
        z: 0.0
    angular_speed_controller:
      angular_gain:
        x: 5.5
        y: 5.5
        z: 5.0 
